.rubocop.yml

```diff
@@ -0,0 +1,129 @@
+AllCops:
+  TargetRubyVersion: 2.3.0
+  Exclude:
+    - "tmp/**/*"
+    - "bin/*"
+    - "spec/dummy/**/*"
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Gemspec/OrderedDependencies:
+  Enabled: false
+
+Layout/AlignParameters:
+  Enabled: true
+  EnforcedStyle: with_fixed_indentation
+Layout/ConditionPosition:
+  Enabled: false
+Layout/DotPosition:
+  EnforcedStyle: leading
+Layout/ExtraSpacing:
+  Enabled: true
+Layout/IndentAssignment:
+  Enabled: False
+Layout/MultilineOperationIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+
+Lint/AmbiguousOperator:
+  Enabled: true
+Lint/AmbiguousRegexpLiteral:
+  Enabled: true
+Lint/DuplicatedKey:
+  Enabled: true
+
+Metrics/ClassLength:
+  Enabled: false
+Metrics/ModuleLength:
+  Enabled: false
+Metrics/AbcSize:
+  Enabled: false
+Metrics/BlockLength:
+  CountComments: true  # count full line comments?
+  Max: 25
+  ExcludedMethods: []
+  Exclude:
+    - "spec/**/*"
+    - "*.gemspec"
+Metrics/CyclomaticComplexity:
+  Enabled: false
+Metrics/LineLength:
+  Max: 80
+Metrics/MethodLength:
+  Enabled: false
+
+Security/Eval:
+  Enabled: true
+  Exclude:
+    - "spec/scenic/schema_dumper_spec.rb"
+Style/BlockDelimiters:
+  Enabled: false
+Style/CollectionMethods:
+  Enabled: true
+  PreferredMethods:
+    find: find
+    inject: reduce
+    collect: map
+    find_all: select
+Style/ConditionalAssignment:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Enabled: true
+  Exclude:
+    - "spec/**/*"
+Style/Documentation:
+  Enabled: false
+Style/FrozenStringLiteralComment:
```

Comment:
We should probably revisit this one in a future change. Just didn't want to do it right now.